index.html

```diff
@@ -144,15 +154,61 @@ <h2 class="title">Uber Navigation</h2>
                     <li class="other-skills">css</li>
                     <li class="other-skills">javascript</li>
                 </ul>
-                <button class="card-btn">See Project</button>
+                <button id="click" class="card-btn" onclick="openPopup()">See Project</button>
             </div>
-        </div>
+        </div>  -->
             </section>
-            <div id="about-section"></div>
+
+ 
+ <!-- <div class="pop"></div> -->
+    
+    <!-- <h2 class="title">Tonic</h2>
+    <ul class="field">
+        <li class="cano">CANOPY</li>
+        <li class="tag">&#8226; Back End Dev </li>
+        <li class="tag">&#8226; 2015</li>
+    </ul>
+    <img src="images/6.png" alt="">
+</div>
+
+    <div class="last-pop">
+        <div class="pop-para">
+    <p>
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+         when an unknown printer took a galley of type and scrambled it 1960s. 
+         Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+    </p><br>
+        <p> 
+        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+        when an unknown printer took a galley of type and scrambled it 1960s 
+        with the releorem Ipsum is simply dummy text of the printing and typesetting industry. 
+        Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn 
+        printer took a galley of type and scrambled it 1960s.
+    </p>
+</div> -->
+<!-- <div class="pop-skills">
+    <ul class="trainning2">
```

Comment:
- [ ] Commented-out codes should be removed from the codebase in order to keep it neat! Please remove the commented codes here 😃